packages/lit-html/src/lit-html.ts

```diff
@@ -327,6 +336,19 @@ export const noChange = Symbol.for('lit-noChange');
 
 /**
  * A sentinel value that signals a ChildPart to fully clear its content.
+ *
+ * The values `undefined`, `null`, and the empty string `''` all render an empty
```

Comment:
This changed in Lit 2. In child expressions, `undefined`, `null`, `''`, and `nothing` all render the same thing now - no nodes at all. In attribute expressions `undefined`, etc., will render an empty string and `nothing` will _remove_ the attribute. In property expressions `nothing` becomes `undefined`.

Reply:
This is incredible info thank you. Will add.